setup.py

```diff
@@ -198,16 +219,31 @@ def run(self):
     version=pyprt_version,
     author=pyprt_author,
     description='Python bindings for the "Procedural Runtime" (PRT) of CityEngine by Esri.',
-    long_description='This API enables the execution of CityEngine rules from within Python applications. It consumes '
-                     'Rule Packages (RPK) authored in CityEngine and writes out the resulting geometry according to a '
-                     'specific file format.',
-    url='https://github.com/Esri/pyprt',
+    long_description_content_type="text/markdown",
+    long_description=long_description,
+    url='https://esri.github.io/pyprt/',
+    download_url='https://github.com/Esri/pyprt/releases',
+    project_urls={"Documentation": "https://github.com/Esri/pyprt/blob/master/README.md#documentation",
+                  "Examples": "https://github.com/Esri/pyprt-examples",
+                  "Source Code": "https://github.com/Esri/pyprt"},
     platforms=['Windows', 'Linux'],
     packages=find_packages(exclude=['tests']),
     include_package_data=True,
     ext_modules=[CMakeExtension('pyprt.pyprt', 'src')],
     cmdclass={'build_ext': CMakeBuild,
               'clean': CleanCommand, 'build_doc': BuildDoc},
+    license="PyPRT is free for non-commercial use. An exception is the PyPRT source code which is licensed under the Apache License, Version 2.0; you may not use this work except in compliance with the License.",

```

Comment:
This sentence still bothers me :) Could it be a copy paste error?

From the main README.md:

> PyPRT is under the same license as the included CityEngine SDK.
> 
> An exception is the PyPRT source code (without CityEngine SDK, binaries, or object code), which is licensed under the Apache License, Version 2.0 (the “License”); you may not use this work except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
> 

Let's use this text with a link to https://github.com/esri/esri-cityengine-sdk on "CityEngine SDK"